### PR TITLE
Update to node 10 in .travis.yml

### PR DESCRIPTION
**Pull request by aikoven**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,7 +1,8 @@
 language: node_js
 
 node_js:
-  - "6"
+  - "9"
+  - "10"
```
Comment (suutari-ai) [inline .travis.yml]: Are you planning to support the latest LTS release of Node.js, i.e. 8? I would appreciate if that was supported and then in the test matrix too of course...

Comment (aikoven) [reply .travis.yml]: Sure!

**Comment by suutari-ai** [comment]
Travis is complaining this:
> tests/index.ts (146,18): Cannot invoke an expression whose type lacks a call signature. Type 'ActionCreator<Failure<{ foo: string; }, any>>' has no compatible call signatures. (2349)

Are you sure that the test is correct? 

The mentioned test line
https://github.com/aikoven/typescript-fsa/blob/7d2b11df281974f14c37b01bb4f7e39b35407383/tests/index.ts#L146
doesn't seem correct to me because the type for errors is specified as `{bar: string}` (on line 129), but the test line passes in a string.

**Comment by aikoven** [comment]
@suutari-ai Fixed in another commit

**Pull request merged**